test_runner/src/test/kotlin/ftl/reports/xml/JUnitXmlTest.kt

```diff
@@ -501,4 +503,32 @@ junit.framework.Assert.fail(Assert.java:50)</failure>
         val oneSuiteXml = parseOneSuiteXml(crashingOneSuiteMessage).xmlToString().trimIndent()
         Assert.assertEquals("One Suite Messages should be the same!", expectedOneSuiteMessage, oneSuiteXml)
     }
+
+    @Test
+    fun `merge only passed nodes`() {
+        val testSuiteWithError = """
+         <testsuite name="" tests="1" failures="1" errors="0" skipped="0" time="2.676" timestamp="2020-05-19T14:40:18" hostname="localhost">
+<properties/>
+<testcase name="test" classname="com.example.test_app.InstrumentedTest" time="1.516">
+<failure>
```

Comment:
Let's use an example from the real XML linked to on the Slack thread from the issue.

https://firebase-community.slack.com/files/UAZSWDV4Z/F013EJLR4K1/2020-05-15_23-04-57.325000_yvmd_nexuslowres-23-en-portrait-test_results_merged.xml

I like taking bug reports and then turning them into tests to ensure they don't regress in the future. 🙂

Reply:
I think starting work from falling test is good idea. For me it can help understand problem and is faster than running full path to reproduce problem every time and as you say we have regress protection. 🙂